broker-daemon/utils/nano-to-datetime.js

```diff
@@ -0,0 +1,17 @@
+const nano = require('nano-seconds')
+
+/**
+ * Converts nanoseconds string to ISO8601 datetime
+ *
+ * @param {String} nanoseconds
+ * @return {String} ISO8601 string
+ */
+function nanoToDatetime (nanoseconds) {
+  const first = nanoseconds.substring(0, nanoseconds.length - 9)
```

Comment:
[nit] Im not _exactly_ sure of a good idea for these variables, but we should try to be more descriptive as `first` and `second` are generic terms. The first half of the string is a millisecond timestamp and the second portion is nanoseconds.